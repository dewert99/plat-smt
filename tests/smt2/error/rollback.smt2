(declare-sort R)
(declare-const p Bool)
(declare-fun q (R) Bool)
(declare-const a R)
(declare-const b R)
(assert (and p (q a) (= a b) bad))
(assert (and (= a b) (q a) p bad))
(check-sat-assuming ((not p)))
(check-sat-assuming ((not (q b))))
(check-sat-assuming ((not (q a))))
(check-sat-assuming ((distinct a b)))