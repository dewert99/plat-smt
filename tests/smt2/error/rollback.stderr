unknown identifier `bad` at 6:30
(assert (and p (q a) (= a b) bad))
                             ^

unknown identifier `bad` at 7:30
(assert (and (= a b) (q a) p bad))
                             ^

